File: battleship.js
let randomLoc = Math.floor(Math.random() * 5);
let location1 = randomLoc;
let location2 = location1 + 1;
let location3 = location1 + 2;
let guess;
let hits = 0;
let isSunk = false;

while (isSunk == false) {
	guess = prompt("Ready, aim, fire! (enter a number from 0-6):");
	if (guess < 0 || guess > 6) {
		alert("Please enter a valid cell number!");
	} else {
		if (guess == location1 || guess == location2 || guess == location3) {
			alert("HIT!");
			hits = hits + 1;
			if (hits == 3) {
				isSunk = true;
				alert("You sank my battleship!");
			}
		} else {
			alert("MISS");
		}
	}
}

// Group  task - port (convert) to Python

File: battleship.py
from random import randint
#import random

randomLoc = randint(1, 6)

location1 = randomLoc
location2 = location1 + 1
location3 = location1 + 2

#guess;
hits = 0;
isSunk = False

while (isSunk == False):
    guess = int(input("Ready, aim, fire! (enter a number from 0-6):"))
    if (guess < 0 or guess > 6):
        print("Please enter a valid cell number!")
    else:
        if (guess == location1 or guess == location2 or guess == location3):
            print("HIT!")
            hits = hits + 1
            if (hits == 3):
                isSunk = True
                print("You sank my battleship!")
        else:
            print("MISS")

